dspace-server-webapp/src/test/java/org/dspace/app/rest/matcher/WorkspaceItemMatcher.java

```diff
@@ -82,6 +82,31 @@ public static Matcher matchItemWithTitleAndDateIssuedAndSubject(WorkspaceItem wi
                 matchLinks(witem));
     }
 
+    /**
+     * Check that the workspace item has the expected type and series values
+     * (used in type bind evaluation)
+     * @param witem the workspace item
+     * @param type  the dc.type value eg. Technical Report
+     * @param series the series value eg. 11-23
+     * @return  Matcher result
+     */
+    public static Matcher matchItemWithTypeAndSeries(WorkspaceItem witem, String type, String series) {
+        return allOf(
+                // Check workspaceitem properties
+                matchProperties(witem),
+                // Check type appears or is null
+                type != null ?
+                        hasJsonPath("$.sections.traditionalpageone['dc.type'][0].value", is(type)) :
```

Comment:
use static final to avoid duplication

Reply:
@vidiecan This is not my code, I copied it from the upstream